infra/modules/network/vpc_endpoints.tf

```diff
@@ -18,6 +18,9 @@ locals {
 
     # AWS services used by ECS Exec
     var.enable_command_execution ? ["ssmmessages"] : [],
+
+    # AWS services used by notifications
+    var.enable_notifications ? ["pinpoint", "email-smtp"] : [],
```

Comment:
It's called `mobile-targeting` [in IAM](https://github.com/navapbc/template-infra/blob/cb15833ca914af837d380ceec3db9aae432dc811/infra/project-config/aws_services.tf#L21) but the VPC endpoints are `["pinpoint", "email-smtp"]` ??? bah

Reply:
yeah i put mobiletargeting at first and got an error saying that doesn't exist.

also, the AWS docs don't mention "email-smtp" anywhere, instead they said to search for smtp and then click on that one, so I had to go to the console and search to find out that email-smtp was what they meant.

Reply:
<img width="664" alt="image" src="https://github.com/user-attachments/assets/cfa2958d-9b02-41a5-b918-76195911735e" />
https://docs.aws.amazon.com/ses/latest/dg/send-email-set-up-vpc-endpoints.html

amazing right?